.changes/unreleased/tinymce-TINY-10589-2024-02-19.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Changed
+body: Change table height resizing handling to remove heights from tds/ths and only apply to trs.
```

some markdown will help a lot, I think
```suggestion
body: Change table height resizing handling to remove heights from `td`/`th` elements and only apply to `tr` elements.
```